datadog_checks_base/datadog_checks/base/utils/db/core.py

```diff
@@ -105,6 +105,9 @@ def execute(self, extra_tags=None):
                         continue
                     elif column_type == 'tag':
                         tags.append(transformer(None, column_value))  # get_tag transformer
+                    elif column_type == 'tag_none':
+                        if column_value:
+                            tags.append(transformer(None, column_value))  # get_tag transformer
```

Comment:
nit - It's not clear to me at first sight that a `tag_none` type means that we are not going to add the tag if the value is none (or empty), but quite the opposite to be honest 😅 However I'm pretty bad at naming things so maybe we could find another name, such as `tag_not_none` or something that states more clearly what it does?

Reply:
I'm also not good at naming things and did not have better idea 😅.
`tag_not_none` may make more sense, I'm gonna change it